services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/models/resource_tracker_container.py

```diff
@@ -1,24 +1,38 @@
+from datetime import datetime
 from typing import Any
 
 from arrow import Arrow
 from models_library.products import ProductName
 from models_library.projects import ProjectID
+from models_library.projects_nodes_io import NodeID
+from models_library.services import ServiceKey, ServiceVersion
 from models_library.users import UserID
 from pydantic import BaseModel
 
+# Scraped from prometheus
 
-class ContainerResourceUsageMetric(BaseModel):
+
+class ContainerScrapedResourceUsageMetric(BaseModel):
     container_id: str
     image: str
     user_id: UserID
+    user_email: str | None
     product_name: ProductName
     project_uuid: ProjectID
+    project_name: str | None
+    node_uuid: NodeID
+    node_label: str | None
+    instance: str | None
     service_settings_reservation_nano_cpus: int | None
     service_settings_reservation_memory_bytes: int | None
     service_settings_reservation_additional_info: dict[str, Any] = {}
```

Comment:
IMO a bit of doc (one sentence description?) on generic fields like `service_settings_reservation_additional_info` or `instance` would help :-)